src/unix/linux_like/linux/mod.rs

```diff
@@ -2404,6 +2421,10 @@ f! {
     pub fn RT_LOCALADDR(flags: u32) -> bool {
         (flags & RTF_ADDRCLASSMASK) == (RTF_LOCAL | RTF_INTERFACE)
     }
+
+    pub fn SO_EE_OFFENDER(ee: *const ::sock_extended_err) -> *mut ::sockaddr {
+        ee.offset(1) as *mut ::sockaddr
+    }
```

Comment:
This would need a test.

Reply:
Would you suggest I add a wrapper around the C macro and test equivalence, like is done for `cmsg`, eg in `test_cmsg_data`?

Reply:
Yes, that would be great!